images/openstack/Dockerfile.ci

```diff
@@ -1,18 +1,17 @@
 # This Dockerfile is used by CI to test using OpenShift Installer against an OpenStack cloud.
 # It builds an image containing the openshift-install command as well as the openstack cli.
-
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/installer
 COPY . .
 RUN hack/build.sh
 
 FROM registry.svc.ci.openshift.org/origin/4.1:base
 COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+COPY --from=builder images/openstack/rdo-stein.repo /etc/yum.repos.d/rdo-stein.repo
```

Comment:
Hmm, shouldn't we use abs path for this?

Reply:
Can we unblock our CI and I will put a follow up to fix this?